.github/workflows/miscellaneous.yml

```
@@ -0,0 +1,30 @@
+name: misc
+
+on: [pull_request]
+
+jobs:
+  # Miscellaneous organizational tests.
+  misc-eof:
+    runs-on: ubuntu-latest
+    container: quay.io/enarx/fedora-test
+    steps:
+    - uses: actions/checkout@v2
+  
+    # Ensure all code-related files have a newline at end of file.
+    - name: All code files have a newline at EOF
+      run: .tests/misc-eof
+
+  # Enforce correct license files and headers.
+  correct-licenses:
```

```suggestion
  misc-licenses:
```

Also, update test file names.

---

Done.